src/ui/public/timepicker/kbn_global_timepicker.html

```diff
@@ -26,15 +26,15 @@
     ng-show="timefilter.isAutoRefreshSelectorEnabled"
     ng-click="kbnTopNav.toggle('interval')"
   >
-    <span ng-show="timefilter.refreshInterval.value === 0">
+    <span ng-show="timefilter.getRefreshInterval().value === 0">
```

Comment:
Since this and other calls to timefilter in this and other templates are not observing the update events emitted by timefilter they are relying on these updates always happening within a digest cycle, but they may or may not be. Since timefilter shouldn't be aware of angular, and it would be even worse if we required everything that mutates the timefilter to be aware of angular (kind of what is happening in this pr right now thanks to things like https://github.com/elastic/kibana/pull/19852/files#diff-caf157a717f917f9deb107d068db88baR24) we should probably be doing something like this in directives that consume the timefilter in their templates:

```js
// in the controller for `<kbn-global-timepicker>`

$scope.$listen(timefilter, 'update', function () {
  // eval async here is just making sure that they callback is executed within
  // a digest cycle, if we somehow ensure that handlers passed to `$scope.$listen()`
  // always execute within a digest cycle this won't be necessary
  $scope.$evalAsync(() => {
    $scope.timefilterValues = {
      refreshInterval: timefilter.getRefreshInterval(),
      time: timefilter.getTime()
    }
  });
});
```

Then within the templates we can reference `timefilterValues.refreshInterval` and `timefilterValues.time` and they will always update correctly.